examples/go/hello-oauth2/hello.go

```diff
@@ -166,6 +166,12 @@ func send(w http.ResponseWriter, req *http.Request) {
 	json.NewEncoder(w).Encode(apps.CallResponse{
 		Markdown: md.MD(message),
 	})
+
+	// Store new token if refreshed
+	tok, err := tokenSource.Token()
+	if err != nil && tok.AccessToken != token.AccessToken {
+		mmclient.AsActingUser(creq.Context).StoreOAuth2User(creq.Context.AppID, tok)
```

Comment:
Nit pick: Naming suggestion
```suggestion
	newToken, err := tokenSource.Token()
	if err != nil && newToken.AccessToken != token.AccessToken {
		mmclient.AsActingUser(creq.Context).StoreOAuth2User(creq.Context.AppID, newToken)
```